src/watts/plugin.py

```diff
@@ -84,6 +86,45 @@ def workflow(self, params: Parameters, name='Workflow') -> Results:
 
         return result
 
+    def convert_unit(self, params: Parameters, unit_system, unit_temperature):
+        """Perform unit conversion
+
+        Parameters
+        ----------
+        params
+            Parameters used when rendering template
+        unit_system
+            Desired unit system: SI or CGS
+        unit_temperature
+            Desired unit for temperature parameter
+
+        Returns
+        -------
+        A copy of params with the converted units
+        """
+        u.imperial.enable()
+        params_copy = copy.deepcopy(params)
+
+        temperature_units = ['Kelvin', 'Celsius', 'Rankine', 'Fahrenheit',
+                            'deg_C', 'deg_R', 'deg_F']
+
+        for key in params_copy.keys():
+
+            if isinstance(params_copy[key], u.quantity.Quantity):
+
+                # Unit conversion for temperature needs to be done separately because 
+                # astropy uses a different method to convert temperature.
+                # Variables are converted to SI by default.
+
+                if params_copy[key].unit in temperature_units:
+                    params_copy[key] = params_copy[key].to(u.Unit(unit_temperature), equivalencies=u.temperature()).value
```

Comment:
Isn't it sufficient to just pass a string as the first argument to the `to()` method? i.e.
```suggestion
                    params_copy[key] = params_copy[key].to(unit_temperature, equivalencies=u.temperature()).value
```